integration-cli/docker_cli_inspect_test.go

```diff
@@ -142,33 +124,28 @@ func (s *DockerSuite) TestInspectTypeFlagWithInvalidValue(c *check.C) {
 	dockerCmd(c, "run", "--name=busybox", "-d", "busybox", "true")
 
 	out, exitCode, err := dockerCmdWithError("inspect", "--type=foobar", "busybox")
-	if exitCode != 0 || err != nil {
-		if !strings.Contains(out, "not a valid value for --type") {
-			c.Fatalf("failed to inspect image: %s, %v", out, err)
-		}
-	}
+	c.Assert(err, checker.NotNil, check.Commentf("%s", exitCode))
+	c.Assert(exitCode, checker.Equals, 1, check.Commentf("%s", err))
+	c.Assert(out, checker.Contains, "not a valid value for --type")
 }
 
 func (s *DockerSuite) TestInspectImageFilterInt(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 	imageTest := "emptyfs"
 	out, err := inspectField(imageTest, "Size")
-	c.Assert(err, check.IsNil)
+	c.Assert(err, checker.IsNil)
 
 	size, err := strconv.Atoi(out)
-	if err != nil {
-		c.Fatalf("failed to inspect size of the image: %s, %v", out, err)
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("failed to inspect size of the image: %s, %v", out, err))
 
 	//now see if the size turns out to be the same
 	formatStr := fmt.Sprintf("--format='{{eq .Size %d}}'", size)
-	out, exitCode, err := dockerCmdWithError("inspect", formatStr, imageTest)
-	if exitCode != 0 || err != nil {
-		c.Fatalf("failed to inspect image: %s, %v", out, err)
-	}
-	if result, err := strconv.ParseBool(strings.TrimSuffix(out, "\n")); err != nil || !result {
-		c.Fatalf("Expected size: %d for image: %s but received size: %s", size, imageTest, strings.TrimSuffix(out, "\n"))
-	}
+	out, exitCode := dockerCmd(c, "inspect", formatStr, imageTest)
+	c.Assert(exitCode, checker.Equals, 0)
```

Comment:
Same as above : Not needed, `exitCode` is never equal to `0` if it gets on this line.